### DISCUSSION TITLE
Can't install 1.3.2

### DISCUSSION THREAD

**Discussion by embraceentropy** [Jyers/Marlin#668] [Q&A] [answered]
I'm currently on 1.3.1 5x5 Manual Mesh 4.2.2. I downloaded the same pre-compiled version of 1.3.2 and have tried several times to flash with no success. With the SD card in the printer boots up as it normally would and loads into the menu. I shut down the printer, remove the card, power on, and the info section still shows 1.3.1. I've tried several times by formatting the SD card, downloading a fresh file, and attempting to install following the instructions in the wiki.

**Comment by smooreace** [comment, marked as answer]
Change the file name

**Reply by embraceentropy** [reply]
Thank you! That worked!

**Comment by rakosi2** [comment]
The release version should really be in the file name so at least something is changing between each release.

**Comment by smooreace** [comment]
That serves little purpose. If you want it names something special then
save it that way when its loaded. A smart person would be maintaining
different versions, and variations of versions, in folders named for the
version number. And with that, the version number IN the file name is not
only a waste but also a significant issue for those running an OS that cant
deal with technically improper file name lengths.

Either way, how did the install go?

**Comment by smooreace** [comment]
Great!
